File: backend/src/main/java/org/example/backend/calculator/model/values/EditValuesModel.java
package org.example.backend.calculator.model.values;

import org.example.backend.calculator.model.*;

import java.util.List;

public record EditValuesModel(
        List<SexMultipliersModel> sexMultipliersValues,
        List<ActivityLevelModel> activityLevelValues,
        List<IsBreastfeedingModel> breastfeedingValues,
        List<IsPregnantModel> pregnantValues,
        List<WeightGoalModel> weightGoalValues
) {
}

File: frontend/src/models/calculator/EditValuesModel.ts
import type { SexMultipliersModel } from "./SexMultipliersModel.ts";
import type { ActivityLevelModel } from "./ActivityLevelModel.ts";
import type { IsBreastfeedingModel } from "./IsBreastfeedingModel.ts";
import type { IsPregnantModel } from "./IsPregnantModel.ts";
import type { WeightGoalModel } from "./WeightGoalModel.ts";

export type EditValuesModel = {
  sexMultipliersValues: SexMultipliersModel[];
  activityLevelValues: ActivityLevelModel[];
  breastfeedingValues: IsBreastfeedingModel[];
  pregnantValues: IsPregnantModel[];
  weightGoalValues: WeightGoalModel[];
};
